compliance_checker/imos/imos.py

```diff
@@ -1311,6 +1311,29 @@ def check_quality_control_conventions_for_quality_control_variable(self, dataset
 
         return ret_val
 
+    def check_quality_control_variable_matches_variable(self, dataset):
+        """
+        Check that the name of a quality control variable which is <DATA>_quality_control
+        matches the name of a corresponding data variable <DATA>.
+        """
+        ret_val = []
+        for qc_variable in self._quality_control_variables:
+            qc_variable_root_name = qc_variable.name[:-16] # '_quality_control' is 16 characters long
+            result_name = ('qc_var', qc_variable.name, 'check_matching_variable_name')
```

Comment:
I've been removing the redundant word `check` from the check names, trying to keep them relatively short. How about `('qc_var', qc_variable.name, 'match_variable_name')`?